modules/themes/gruvbox-dark.nix

```diff
@@ -1,11 +1,20 @@
-{ config, lib, ... }:
+{ config, lib, pkgs, ... }:
 
 with lib;
 
 {
   config.soxin.themes.gruvbox-dark = {
-    rofi = {
-      name = "gruvbox-dark";
+    neovim = {
+      plugins = singleton {
+        plugin = with pkgs.vimPlugins; gruvbox-community;
+        config = ''
+          set background=dark
+          colorscheme gruvbox
+          let g:airline_theme='gruvbox'
```

Comment:
Does that imply that we will always ship the `airline` plugin?

Reply:
This does not mean airline is assumed installed. It's just a global variable that is a no-op without airline but does apply if you have it installed.

Reply:
Great then!